ee/urls.py

```diff
@@ -35,7 +35,6 @@ def extend_api_router(
 ) -> None:
     root_router.register(r"billing", billing.BillingViewset, "billing")
     root_router.register(r"license", license.LicenseViewSet)
-    root_router.register(r"time_to_see_data", time_to_see_data.TimeToSeeDataViewSet, "query_metrics")
```

Comment:
this route wasn't for capture of the data, that goes to `/insight/timing`, this was just for querying